api/src/main/java/com/codeforcommunity/dto/user/UserDataResponse.java

```diff
@@ -8,27 +8,41 @@
   private String firstName;
   private String lastName;
   private String email;
+  private Integer id;
   private Country country;
   private PrivilegeLevel privilegeLevel;
+  private Boolean disabled;
 
   public UserDataResponse(
       String firstName,
       String lastName,
+      Integer id,
       String email,
       Country country,
-      PrivilegeLevel privilegeLevel) {
+      PrivilegeLevel privilegeLevel,
+      Boolean disabled) {
```

Comment:
⛏️ Would prefer `isDisabled`

Reply:
Sounds good, will make the change @rymaju